app/pages/lab-pages-editor/components/TasksPage.jsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-console */
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/require-default-props */
+/* eslint-disable radix */
+
+import { useWorkflowContext } from '../context.js';
+import strings from '../strings.json'; // TODO: move all text into strings
+
+import GripIcon from '../icons/GripIcon.jsx';
+
+export default function TasksPage() {
+  const { workflow, update } = useWorkflowContext();
+  const isActive = true; // TODO
+
+  function placeholderEventHandler() {
+    console.log('+++ TODO');
+  }
+
+  console.log('+++ workflow: ', workflow);
+
+  if (!workflow) return null;
+
+  return (
+    <div className="tasks-page">
+      <div className="workflow-title flex-row">
+        <h2 className="flex-item">{workflow?.display_name}</h2>
+        <span className="workflow-id">{`#${workflow?.id}`}</span>
+        {(isActive) ? <span className="status-active">Active</span> : <span className="status-inactive">Inactive</span>}
+      </div>
+      <section aria-labelledby='workflow-tasks-heading'>
+        <h3 id='workflow-tasks-heading'>Tasks</h3>
+        <div className="flex-row">
+          <button
+            className="flex-item big primary"
+            onClick={placeholderEventHandler}
+            type="button"
+          >
+            Add a new Task +
+          </button>
+          {/* Dev observation: the <select> should have some label to indicate it's for choosing the starting task. */}
+          <select
+            aria-label="Choose starting test"
+            className="flex-item"
+          >
+            <option disabled={true}>Choose starting test</option>
```

Comment:
```suggestion
            <option disabled >Choose starting test</option>
```
You don't specify the `true` or `false` for boolean props in React.